### ISSUE TITLE
DiscordSRV API hook - NoSuchMethodError

### ISSUE THREAD

**Issue by mibby** [kangarko/ChatControl-Pro#1387]
ChatControlPro v8.3.14
DiscordSRV v1.17.2 (dev branch compiled as of commit https://github.com/DiscordSRV/DiscordSRV/commit/b86608be80719ee81a233c827ced2dda765a415e)

@kangarko ChatControlPro breaks with newer versions of DiscordSRV.

```
[02:27:01] [Async Chat Thread - #0/ERROR]: Could not pass event AsyncPlayerChatEvent to ChatControl v8.3.14
java.lang.NoSuchMethodError: github.scarsz.discordsrv.DiscordSRV.config()Lorg/bukkit/configuration/file/FileConfiguration;
	at org.mineacademy.chatcontrol.lib.fo.model.lPt6.const(SourceFile:2589) ~[?:?]
	at org.mineacademy.chatcontrol.lib.fo.model.CoM3.const(SourceFile:1284) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .A.const(SourceFile:158) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .A.const(SourceFile:86) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .A.const(SourceFile:33) ~[?:?]
	at org.mineacademy.chatcontrol.final.nul. .LPt5.execute(SourceFile:77) ~[?:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[patched_1.14.4.jar:git-Paper-186]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.14.4.jar:git-Paper-186]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:545) ~[patched_1.14.4.jar:git-Paper-186]
	at net.minecraft.server.v1_14_R1.PlayerConnection.chat(PlayerConnection.java:1713) ~[patched_1.14.4.jar:git-Paper-186]
	at net.minecraft.server.v1_14_R1.PlayerConnection.a(PlayerConnection.java:1628) ~[patched_1.14.4.jar:git-Paper-186]
	at net.minecraft.server.v1_14_R1.PacketPlayInChat$1.run(PacketPlayInChat.java:41) ~[patched_1.14.4.jar:git-Paper-186]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_222]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_222]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_222]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
```

**Comment by kangarko** [comment]
Thanks, should be fixed but havent tested. We will only support the officially released version on SpigotMC.

**Comment by mibby** [comment]
Sure, that's fine. Just giving you a heads up though since I do test the latest development builds for future DiscordSRV releases.

**Comment by kangarko** [comment]
Alright, I am very confident in that the last release should now work with the latest dev of DiscordSRV let me know if not!

**Comment by mibby** [comment]
@kangarko Heads up, I think DiscordSRV integration may be broken with ChatControlPro 8.3.17 and DiscordSRV 1.17.2-SNAPSHOT. DiscordSRV correctly starts up and feeds player logins / server status to my in-game channel, but chat does not forward and neither does staff chat.

Downgrading to ChatControlPro 8.3.16 and DiscordSRV 1.17.1, chat is properly forwarding again. So perhaps the hook is broken if there are API changes?

Settings.yml
```
Admin_Chat:
  Bungee: true
  Log_To_Console: true
  Write_To_File: true
  Discord_Channel: staff-in-game
  Shortcut: none
Bungee_Chat:
  Enabled_By_Default: true
  Log_To_Console: true
  Write_To_File: false
  Discord_Channel: in-game
  Shortcut: '!'
```

Formatting.yml
```
Integration:
  Discord_Channel: 'in-game'
```

DiscordSRV's config.yml
```
Channels: {"in-game": "280012581983354881", "staff-in-game": "494635905013907457"}
RespectChatPlugins: true
```

**Comment by kangarko** [comment]
Thanks please try the next version.

**Comment by mibby** [comment]
Apologies for the late rely -- seems to be fixed with v8.3.18. Thanks!
